tests/src/python/test_qgsgeometry.py

```diff
@@ -1305,7 +1305,9 @@ def testVertexAt(self):
         polyline = QgsGeometry.fromPolylineXY(points)
 
         for i in range(0, len(points)):
-            self.assertEqual(QgsPoint(points[i]), polyline.vertexAt(i), "Mismatch at %d" % i)
+            # WORKAROUND to avoid a system error
```

Comment:
This looks scary -- what happens here?

Reply:
I'll retry but it's not the first time I have encountered this kind of problem. Maybe a problem with SIP?

Reply:
@nyalldawson OK, it's come from the newest signature of QgsPoint.
My previous comment: https://github.com/qgis/QGIS/pull/9645#issuecomment-477904611